styles/light/_light-theme.scss

```
@@ -1181,3 +1181,34 @@ main article .card-grid {
     background-color: $gray-200;
   }
 }
+
+.page-tokenization {
+  .project-name {
+    color: $white;
+  }
```

Since I got the go-ahead from design to make light cards here, we don't need this rule anymore, but we do need a new rule to invert the logos instead.

```suggestion
  .project-cards .project-logo {
    filter: invert(100%);
  }
```